t/t7527-internal-fsmonitor.sh

```diff
@@ -564,7 +564,7 @@ do
 			'
```

Comment:
There are a lot of lines in this file that do `git config --unset core.fsmonitor` to unset the `:internal:` value.  We need to change them too.